OneDrive returns Null for WebUrl

In some situations OneDrive omits returning the WebUrl field, which results in crashing applications.

```
Caused by: java.lang.NullPointerException: Cannot invoke "String.toString()" because "this.url" is null
	at ch.cyberduck.core.DescriptiveUrl.getUrl(DescriptiveUrl.java:75) ~[core-9.1.6.jar:53bf5bba71249806e3fe4dec04be6db3a15355b0]
	at ch.cyberduck.core.PathAttributes.serialize(PathAttributes.java:242) ~[core-9.1.6.jar:53bf5bba71249806e3fe4dec04be6db3a15355b0]
…
	at ch.cyberduck.core.onedrive.AbstractListService.list(AbstractListService.java:66) ~[onedrive-9.1.6.jar:53bf5bba71249806e3fe4dec04be6db3a15355b0]
	at ch.cyberduck.core.onedrive.OneDriveListService.list(OneDriveListService.java:53) ~[onedrive-9.1.6.jar:53bf5bba71249806e3fe4dec04be6db3a15355b0]
```

--- a/onedrive/src/main/java/ch/cyberduck/core/onedrive/features/GraphAttributesFinderFeature.java
+++ b/onedrive/src/main/java/ch/cyberduck/core/onedrive/features/GraphAttributesFinderFeature.java
@@ -28,6 +28,7 @@ import ch.cyberduck.core.onedrive.GraphExceptionMappingService;
 import ch.cyberduck.core.onedrive.GraphSession;
 import ch.cyberduck.core.webloc.UrlFileWriterFactory;
 
+import org.apache.commons.lang3.StringUtils;
 import org.nuxeo.onedrive.client.OneDriveAPIException;
 import org.nuxeo.onedrive.client.types.DriveItem;
 import org.nuxeo.onedrive.client.types.DriveItemVersion;
@@ -50,7 +51,12 @@ public class GraphAttributesFinderFeature implements AttributesFinder, Attribute
     }
 
     static Optional<DescriptiveUrl> getWebUrl(final DriveItem.Metadata metadata) {
-        return Optional.of(new DescriptiveUrl(metadata.getWebUrl(), DescriptiveUrl.Type.http));
+        final String webUrl = metadata.getWebUrl();
+        if(StringUtils.isBlank(webUrl)) {
+            return Optional.empty();
+        }
+
+        return Optional.of(new DescriptiveUrl(webUrl, DescriptiveUrl.Type.http));
     }
 
     @Override

